### DISCUSSION TITLE
Save favorite modules

### DISCUSSION THREAD

**Discussion by pablobaico** [DISTRHO/Cardinal#569] [Q&A] [answered]
I am using version 23.07 and I see that the ability to save or mark modules as "favorites" does not work. They can be marked, but when you close the session and reload the plugin they do not appear.
I have tried it in all the variants of the plugin, in VST and CLAP format, and also in its native version, with the same result.
Could it be a function to be implemented in the future or is it a bug?

Thank you.

**Comment by dromer** [comment, marked as answer]
This has been answered many times already: https://github.com/DISTRHO/Cardinal/issues?q=is%3Aissue+favorites+is%3Aclosed

**Reply by falkTX** [reply]
except the situation has changed, and favorites are meant to be global state now instead of per project.
seems we still save favorites on a per-project basis, which leads to it overwriting preferences on project load.

**Reply by pablobaico** [reply]
Thanks.
